### PR TITLE
SUP-8534 passing KS to the player using SessionInfo service

### PR DESCRIPTION
**Pull request by omridevk**

### CODE REVIEW

**Comment by omridevk** [comment]
@atarsh 

Can you please review and let me know if there's a way for me to verify the solution?

Not sure at the moment if it actually works.

**Comment by atarsh-zz** [comment]
@omridevk  it look ok, and the repo's readme explains how to run the project locally.

**Comment by omridevk** [comment]
@atarsh 
Tested locally, works as expected.

I have noticed a bug if you have Apply Access Control on thumbnails, we are not adding the KS to the thumbnail, so it does not load.

**Comment by atarsh-zz** [comment]
@omridevk feel free to open a bug and fix it :)

**Comment by omridevk** [comment]
done already.

**Comment by omridevk** [comment]
oops sorry. need to recommit without those files. will fix

File: app/index.html [hunk 1]
```diff
@@ -27,18 +27,18 @@
...
-	
+
 	window.parent.kmc.vars = {
 
-		ks : 'YOUR_KS',
-		partner_id : 'YOUR_PID',
-		service_url : 'YOUR_SERVICE_URL',
-		cdn_host : 'YOUR_CDN_HOST',
-		liveanalytics : {player_id : 'YOUR_PLAYER_UICONF_ID'}
+		ks : 'djJ8MTc2MzMyMXy1PmA3FtA4jIUlDKrGivuTVPvHXOVcAbsqefa1_5-veLaA4IHmHT-Aq_gq09nQwk6bvGJCG8RYWOBu38pvv_kGO80F8oAUabKtvvLhoClkYXmzG643OZhUWxbeG1lFkz0k0SfaXAZCY1NuR2GaBg_8',
```
Comment (atarsh-zz) [inline app/index.html:33]: LOOOOOOO!!!
please return the placeholders instead of actual data, and make sure to never commit real data here.

**Comment by omridevk** [comment]
I know I know
Mistake. sorry
Recommiting.

**Pull request closed without merging**